easysid.cpp

```diff
@@ -204,7 +220,7 @@ EXPORT void run_ms(uint32_t ms)
     if (pcm == NULL) {
         pcm = makePcmOutputAudio();
     }
-    const size_t buflength = 16384;
+    const size_t buflength = 32768;
```

Comment:
What's this change for? It doesn't seem to be related to validation of input parameters. What's the justification here?